src/sentry/integrations/jira_server/actions/create_ticket.py

```diff
@@ -0,0 +1,34 @@
+from __future__ import annotations
+
+from typing import Any
+
+from sentry.integrations.jira_server.actions.form import JiraServerNotifyServiceForm
+from sentry.rules.actions import TicketEventAction
+from sentry.services.hybrid_cloud.integration import RpcIntegration
+from sentry.utils.http import absolute_uri
+
+
+class JiraServerCreateTicketAction(TicketEventAction):
+    id = "sentry.integrations.jira_server.notify_action.JiraServerCreateTicketAction"
+    label = "Create a Jira issue in {integration} with these "
+    ticket_type = "a Jira issue"
+    link = "https://docs.sentry.io/product/integrations/issue-tracking/jira/#issue-sync"
```

Comment:
I think this should link to [here](https://docs.sentry.io/product/integrations/issue-tracking/jira/#automatically) (and once merged, the docs should be updated to remove that blurb about automatic ticket creation only being available in Jira cloud).

Reply:
We use the link field for the sentence below in the Jira alert rule action settings so I think it makes more sense to keep it as the previous one? I copied it over originally from the Jira action. Also I'll make a ticket to remember to remove that blurb
<img width="608" alt="Screenshot 2023-10-16 at 1 52 25 PM" src="https://github.com/getsentry/sentry/assets/67301797/5141f136-43fd-4866-866d-fa45f5a337da">